### ISSUE TITLE
Post-processing SQL not running - pgosm-meta.sql

### ISSUE THREAD

**Issue by rustprooflabs** [rustprooflabs/pgosm-flex#235]
## What version of PgOSM Flex are you using?

v0.4.5

## Docker image

Built locally from 0.4.5 tag.

## What operating system, osm2pgsql, and PostgreSQL/PostGIS versions are you using?

N/A

## What did you do exactly?

I was working on #226 and ran into a failure because the replication related procedures created in `pgosm-meta.sql` were not found. Upon investigation I verified that the output related to post-processing SQL was missing and the comments on tables, columns, etc. were not being added.

## What did you expect to happen?

I expected the post-processing to work.  I expect that if it fails for any reason the operator should be informed of the error.  In the case the error is in the Docker image (like this issue) the `make` command should fail.

## What did you do to try analyzing the problem?

Working on a fix.  Adjusting the `subprocess.run()` call seems to be the trick.  Should add a test to the `Makefile` to catch this error to prevent from happening again.

**Comment by rustprooflabs** [comment]
Limited to the `pgosm-meta.sql` script.  Typo...

**Comment by rustprooflabs** [comment]
Commit acac426 fixes the error in the SQL script causing the issue.

Commits 6554d0e and e8f14a9 improves the post-processing to track errors by SQL script to report failures in post-processing. When this error is detected it is reported as having an error.

```
Post-processing pgosm-meta
Error in post-processing layerset: pgosm-meta
Post-processing amenity
Post-processing building
```

The final output flips to a warning to make the failure obvious and to point the user to the output above.

```
WARNING:pgosm-flex:pgosm_flex:PgOSM Flex completed with errors. Details in output
```

**Comment by rustprooflabs** [comment]
This is resolved in #226.
